userspace/libsinsp/test/events_file.ut.cpp

```diff
@@ -571,3 +571,1127 @@ TEST_F(sinsp_with_test_input, signalfd4)
 	ASSERT_EQ(fdinfo->get_ino(), 0);
 	ASSERT_FD_GETTERS_NOT_FILE(fdinfo)
 }
+
+TEST_F(sinsp_with_test_input, fspath_field_mkdir)
```

Comment:
rename this cpp file to `events_fspath.ut.cpp`. Also thanks for the awesome test coverage 🚀 !

Reply:
I agree with Melissa. They are so many that maybe creating a standalone `events_fspath.ut.cpp` would be good for readability. Also, almost all the following test cases all do the same thing basically, just for different event types. Is there a way of having a loop/template/macro setup that would avoid replicating the same test 15-20 times?

Reply:
Sure I will move all the fspath tests to their own file. I also cleaned up the tests to be more dry--all the various arguments are in one location and there are helper methods for the common patterns of enter events/failed exit events not having any fspath values, as well as helpers to more succinctly describe the expected values for each test case.